samples/step-by-step/Core_5/Client/Program.cs

```diff
@@ -3,24 +3,39 @@
 
 class Program
 {
-
+    #region StepByStep-ClientInit
     static void Main()
     {
+        // This makes it easier to tell console windows apart
```

Comment:
Why does the snippet need to be clean exactly? Aren't comments supposed to explain "why"? What would not be clean would be 6 lines of "I don't have any idea what this does and I have to take this on faith" so of course we have to explain it in some way, at least a little bit.

If that explanation is outside the snippet then you have to repeat everything to refer back. Or you have to divide it into 6 different 1-line snippets, in which case you lose a feel for what it's supposed to look like all together.